src/automation.ts

```diff
@@ -19,15 +19,26 @@ const init = async (workflows: Workflow[], options: Options) => {
   setConfig(options)
 }
 
-const buildEmailAction = (emails: EmailAction[], sendTo: string) => {
+const buildEmailAction = (emails: EmailAction[], sendTo: string, params?:BodyParam[]) => {
   emails.forEach(async ({ sendDelay, subject, body, from }) => {
     const template = 'default'
     const today = moment.utc()
     const sendAt = today.add(sendDelay, 'hours').toDate()
-
+    if (typeof params !== "undefined") {
+      body=setBodyParameters(body,params)
```

Comment:
We should not change the incoming parameter directly. Instead, we can define a new variable like so:
```
let formattedBody = body;
formattedBody = setBodyParameters(body,params);
```

Reply:
I've change it to this :
`
let formattedBody = body;
    if (typeof params !== "undefined") {
      formattedBody=setBodyParameters(body,params)
    }
    await addScheduledMessage({ template, sendFrom, sendTo, sendAt, subject, body:formattedBody })
`